### ISSUE TITLE
Not getting block information for a transaction

### ISSUE THREAD

**Issue by CMarinacci** [MetacoSA/QBitNinja#80]
I was using the NuGet package for QBitNinja and calling QBitNinjaClient.GetTransaction to get transaction information. For some transactions, there is no block information, even though I can see the block information for the transaction using [BlockCypher.com](https://live.blockcypher.com/btc-testnet/tx/d01f564f4f3116bd9cfc18dd74973fe39e80b56cd8983c7503d436fec0791f64/).

I used tapi.qbit.ninja and there is no block information for the transaction there either, using [this URL](http://tapi.qbit.ninja/transactions/d01f564f4f3116bd9cfc18dd74973fe39e80b56cd8983c7503d436fec0791f64).

I'm trying to use QBitNinja to monitor when transactions first get confirmed. Is there a legitimate reason that there is no block information for this transaction? If so, is there a way to do this other than checking the block information?

Thanks,

Chris

**Comment by NicolasDorier** [comment]
Yes I think testnet qbit crashed because I did not updated to 0.16.3. Checking it.

**Comment by NicolasDorier** [comment]
My testnet is currently reindexing. I expect this to be alive in a day or so.

**Comment by NicolasDorier** [comment]
probably need to wait indexing.

**Comment by CMarinacci** [comment]
Okay, thanks!

**Comment by CMarinacci** [comment]
Is the reindexing finished? I'm still not seeing the block information.

**Comment by NicolasDorier** [comment]
yep sorry, testnet qbit was stuck should be coming back soon now.

**Comment by CMarinacci** [comment]
Yes, I'm seeing the block information now.

Thanks!
